etc/teams-for-linux.profile

```diff
@@ -0,0 +1,27 @@
+include disable-common.inc
+include disable-devel.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include globals.local
+
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6,netlink
+seccomp
+
+private-bin sh,xdg-mime,tr,sed,echo,head,cut,xdg-open,grep,egrep,bash,zsh,teams-for-linux
+private-dev
+private-etc fonts,machine-id,localtime,ld.so.cache,ca-certificates,ssl,pki,crypto-policies,resolv.conf
+private-tmp
+disable-mnt
```

Comment:
Move over private-bin.

Reply:
This was the only comment I wasn't sure about what was meant.  When you mention move over private-bin, what do you mean?  Thanks again for all your help in looking at this.

Reply:
This order:
```
disable-mnt
private-bin
private-cache
private-dev
private-etc
private-tmp
```